src/bin/cli.ts

```diff
@@ -187,6 +187,7 @@ const createRun = ({ shouldThrow }) => async function run (argv) {
       err.errors.forEach((err) => console.error(chalk`{red ${err}}\n\n`))
       await Promise.all(serverErrorsWritten)
       console.error(`Please check the errors log for more details: ${errorsFile}`)
+      if (shouldThrow) throw err
```

Comment:
This should be `terminate(err)` like in other [lines](https://github.com/contentful/contentful-migration/blob/master/src/bin/cli.ts#L112)

Reply:
Good point!